tools/data-handler/src/move.ts

```diff
@@ -244,32 +249,35 @@ export class Move {
     await this.rankCard(cardKey, children[index - 1].key);
   }
 
+  /**
+   * Ranks card first.
+   * @param cardKey card key
+   */
   public async rankFirst(cardKey: string) {
-    const card = await this.project.findSpecificCard(cardKey, {
+    const card = await this.getCard(cardKey, {
       metadata: true,
       parent: true,
     });
-    if (!card) {
-      throw new Error(`Card ${cardKey} not found from project`);
-    }
 
     const children = sortItems(
-      await this.getChildren(card.parent || ROOT),
+      await this.getChildren(card),
       (item) => item.metadata?.rank || EMPTY_RANK,
     );
 
     if (!children || children.length === 0) {
-      throw new Error(`Children not found from card ${card.parent}`);
+      // no children --> go first
```

Comment:
The reason why it threw an error before was because it is not possible that there would be no children. We are fetching the children of the card's parent ie. it should always have at least the card itself.

Reply:
I'll revert this row.